website/next.config.js

```diff
@@ -17,7 +17,7 @@ const withNextra = nextra({
   transform(result, { route }) {
     if (route && !result.includes('getStaticProps')) {
       const banner = `
-import { getPageMap } from '@/components/get-page-map'
+import { getPageMap } from '@/src/get-page-map'
```

Comment:
since now `components` folder contain `utils` files `get-page-map` I renamed this folder to just `src`

Reply:
@B2o5T – Why the kebab case? I feel like it should be `getPageMap.ts`.